talk/morelanguage/raii.tex

```diff
@@ -253,7 +253,7 @@
       observe(*pt);                     // Keep ownership
       modifyRef(*pt);                   // Keep ownership
       modifyPtr(pt.get());              // Keep ownership
-      consume(std::move(pt));           // Transfer ownership
+      consumer(std::move(pt));          // Transfer ownership
```

Comment:
Since I renamed all the functions recently and forgot it here: The purpose was to use a verb rather than a noun as function name. So could you please rename the function to `consume` please? Thx!